documentation/user_docs/docs/user_guide/Advanced_use.rst

```diff
@@ -2,22 +2,29 @@
 Advanced use
 ############
 
-The following details more comprehensively the functions in the Horace suite. In the :ref:`Getting started <user_guide/Getting_started:Getting started>` section a brief tutorial was provided on the use of the essential Horace functions. In addition there is also a :ref:`list of functions <manual/list_of_functions:list of functions>`, which details the syntax for every function.
+The following details more comprehensively the functions in the Horace suite. In the :ref:`Getting started
```

Comment:
while this is correct english, I find it more natural to read details as a noun than a verb, and then it looks odd. Maybe provides more comprehensive details....